src/Raven.Studio/typescript/components/pages/database/settings/remoteAttachments/remoteAttachmentsConstants.ts

```diff
@@ -0,0 +1,5 @@
+type DestinationProvider = "s3" | "azure";
+
+const destinationProviderList: DestinationProvider[] = ["s3", "azure"];
+
+export const remoteAttachmentsConstants = { destinationProviderList };
```

Comment:
```suggestion
const destinationProviderList = ["s3", "azure"] as const;

export type DestinationProvider = (typeof destinationProviderList)[number];

export const remoteAttachmentsConstants = { destinationProviderList };
```
but the type is not exported in your case so you can drop it